src/components/BlipGroupCard.vue

```diff
@@ -168,6 +169,20 @@ export default {
       groups.push(group)
       return groups
     }
+  },
+  methods: {
+    isFailedMessageGroup(message, group) {
+      if (message.type === MessageTypesConstants.CONVERSATION_SUMMARY) {
+        return ''
+      }
+      return message.status === 'failed' && message.position === 'right' && group.hasNotification ? ' failed-message' : ''
+    },
+    failedMessageNotification(type) {
+      if (type === MessageTypesConstants.CONVERSATION_SUMMARY) {
+        return this.translations.failedToLoadConversationSummary
```

Comment:
Shouldn't it be the responsibility of the `ConversationSummary` component to display this `failedToLoadConversationSummary`?